### ISSUE TITLE
Capturing completed event

### ISSUE THREAD

**Issue by afshin2003** [OptimalBits/bull#256]
Hi Guys,
I am using two node processes to produce and consume jobs. I am not able to capture completed event in job producer process although it can be done in consumer. Is it by design?

BEST,
Afshin

**Comment by manast** [comment]
yes it is by design, but I am working on a better design where events are global instead of local.

**Comment by afshin2003** [comment]
Manuel,
Thanks for your reply. So I am looking forward for this enhancement :)

Afshin

**Comment by GeoffreyPlitt** [comment]
+1

**Comment by manast** [comment]
I am already working on a solution for this. Expect a few days before completion.
